misc/graylog.conf

```
@@ -518,6 +517,7 @@ message_journal_enabled = true
 #   If you create a separate partition for the journal files and use a file system creating directories like 'lost+found'
 #   in the root directory, you need to create a sub directory for your journal.
 #   Otherwise Graylog will log an error message that the journal is corrupt and Graylog will not start.
+# Default: <data_dir>/journal
 message_journal_dir = data/journal
```

```suggestion
#message_journal_dir = data/journal
```

Couldn't we comment that out?